AEPLifecycle/Sources/Lifecycle.swift

```diff
@@ -145,4 +164,17 @@ public class Lifecycle: NSObject, Extension {
 
         return TimeInterval(sessionTimeoutInt)
     }
+
```

Comment:
Do you think it will be better to keep isInstall() and persistInstallDate() in AudienceState as package protected and access it from state where we have isUpgrade() as well, so we have all the methods in the same place.

Reply:
I would prefer not to include them in LifecycleState which is used for standard workflows and does not have other common code for the two.